coding-concepts.js

```diff
@@ -8,39 +8,39 @@
 // --------------------1) What will this log?
 
```

Comment:
You crushed this section as expected because you showed that you have a great handle on concepts in the test section. Great job!